cmake/Modules/MinetestFindIrrlichtHeaders.cmake

```diff
@@ -1,21 +1,13 @@
-# Locate Irrlicht or IrrlichtMt headers on system.
+# Locate IrrlichtMt headers on system.
 
-foreach(libname IN ITEMS IrrlichtMt Irrlicht)
-	string(TOLOWER "${libname}" libname2)
-
-	find_path(IRRLICHT_INCLUDE_DIR NAMES irrlicht.h
-		DOC "Path to the directory with IrrlichtMt includes"
-		PATHS
-		/usr/local/include/${libname2}
-		/usr/include/${libname2}
-		/system/develop/headers/${libname2} #Haiku
-		PATH_SUFFIXES "include/${libname2}"
-	)
-
-	if(IRRLICHT_INCLUDE_DIR)
-		break()
-	endif()
-endforeach()
+find_path(IRRLICHT_INCLUDE_DIR NAMES irrlicht.h
```

Comment:
seems unlikely anyone would want to install irrlichtmt as a system-wide package. 
How about just removing this file, and always using lib/irrlichtmt from the top-level CMakeLists.txt

Reply:
It does happen, e.g. with distros that package IrrlichtMt separately or Docker builds that don't mind installing everything system-wide.

Reply:
OK... but since the interface for IrrlichtMt is not stable (it just changed a lot!), it would be good to discourage this as much as possible. Accidentally linking or loading the wrong version could cause hard to understand crashes. And since the only user of IrrlichtMt is minetest, there's no benefit to making it a system-wide package or dynamically linked.

Everything else looks good.